.github/workflows/greetings,yml

```diff
@@ -0,0 +1,24 @@
+name: 'Greetings'
+
+on:
+  fork:
+  push:
+    branches: [main]
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: EddieHubCommunity/gh-action-community/src/welcome@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Congratulations, @${{ github.actor }}! 🎉 Thank you for creating your issue. Your contribution is greatly appreciated and we look forward to working with you to resolve the issue. Keep up the great work!
+          pr-message: |
+            Great job, @${{ github.actor }}! 🎉 Thank you for submitting your pull request. Your contribution is valuable and we appreciate your efforts to improve our project.
+          footer: 'We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/Open-Source-Chandigarh/Euphoria-Check/blob/main/CONTRIBUTE.md)'
```

Comment:
[contributing guidelines](https://github.com/Open-Source-Chandigarh/Euphoria-Check/blob/main/CONTRIBUTE.md)
Change the link.

Reply:
I am facing some issue, will create a fresh PR soon, closing this for now.